certified-connectors/App Power Forms/apiDefinition.swagger.json

```diff
@@ -0,0 +1,647 @@
+      "post": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
+          }
+        },
+        "operationId": "AddCardFieldSchema",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "fieldType",
+            "in": "body",
+            "required": false,
+            "schema": {}
+          },
+          {
+            "name": "formName",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "fieldName",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/GetCardResponse/{instanceId}/{name}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "x-ms-dynamic-schema": {
+                "operationId": "GetResponseSchema",
+                "parameters": {
+                  "cardName": {
+                    "parameter": "name"
+                  }
+                }
+              },
+              "x-ms-dynamic-properties": {
+                "operationId": "GetResponseSchema",
+                "parameters": {
+                  "cardName": {
+                    "parameterReference": "name"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "operationId": "GetCardResponse",
+        "summary": "Wait for form response",
+        "description": "Use this action to wait until a specific form is submitted. You can use this action for both Easy Forms and forms based on Adaptive Card JSON.",
+        "parameters": [
+          {
+            "name": "instanceId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "ID of the card to wait for",
+            "x-ms-summary": "Card ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "name",
+            "required": true,
+            "type": "string",
+            "in": "path",
+            "x-ms-summary": "Card name",
+            "x-ms-url-encoding": "single",
+            "description": "Name of the card providing the schema.",
+            "x-ms-dynamic-values": {
+              "operationId": "MetadataListOfCards",
+              "value-path": "name",
+              "value-title": "displayName",
+              "parameters": {
+                "cardType": "ALL"
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/GetResponseSchema/{cardName}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "No need to specify response schema",
+            "schema":{}
```

Comment:
@TerhoAntila Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.

Reply:
No parse JSON required.
![image](https://user-images.githubusercontent.com/10154416/152111622-74904033-c6cf-41d3-a142-017466a90e27.png)

Reply:
Resolution committed.